script.js

```diff
@@ -25,3 +25,206 @@ const LiInMenu = document.getElementsByClassName('menue');
+    A daily selection of privately personalized reads; no accounts or
+    sign-ups required. Has been the industry's standard.
+  </p>`,
+    technology: `  <ul class="button">
+    <li>HTML</li>
+    <li>Bootstrap</li>
+    <li>Ruby</li>
+  </ul>`,
+    button: '  <button class="see">See project</button>',
+  },
+];
+const insertion = function (title, technology, description, button) {
+  const projectSection = document.querySelector('.projects-section');
+  const section = document.createElement('div');
+  projectSection.appendChild(section);
+  section.className = 'data-dashbord';
+
+  section.innerHTML = `
+${title}
+ ${description}
+${technology}
+${button}
+`;
+};
+for (let i = 0; i < 6; i += 1) {
+  insertion(
+    array[i].title,
+    array[i].technology,
+    array[i].description,
+    array[i].button,
+  );
+  if (i === 0) {
+    const dataDash = document.querySelector('.data-dashbord');
+    dataDash.style.background = `url(${array[i].image})`;
+    dataDash.style.paddingBottom = '0px';
+    const firstCardButton = document.querySelector('.data-dashbord button.see');
+    firstCardButton.style.display = 'inline';
+  }
+}
+
+const buttonSee = document.querySelectorAll('.see');
+for (let i = 0; i < 7; i += 1) {
+  buttonSee[i].addEventListener('click', () => {
+    let dynmicTitle = '';
+    let dynamicImage = " <img src='images/availiability.png'>";
+    switch (i) {
+      case 2:
+        dynmicTitle = 'Data Dashbord HealthCare';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 3:
+        dynmicTitle = 'Website portfolio';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 4:
+        dynmicTitle = 'Professional Art Printing Data More';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 5:
+        dynmicTitle = 'Data Dashbord HealthCare';
+        dynamicImage = " <img src='images/availiability.png'>";
+        break;
+      case 6:
+        dynmicTitle = 'Professional Art Printing Data More';
+        dynamicImage = "<img src='images/availiability.png'>";
+        break;
+      case 0:
+        dynmicTitle = 'Multi-post Stories';
+        dynamicImage = "<img src='images/woman.png'>";
+        break;
+      case 1:
+        dynmicTitle = 'Professional Art Printing Printing Data';
+        dynamicImage = '   <img src="images/professional.png">';
+        break;
+      default:
+        dynamicImage = 'no image';
+        dynmicTitle = 'no title';
+    }
+    const divParent = document.createElement('div');
+    const popUp = document.createElement('div');
+    popUp.className = 'popUpParent';
+    popUp.innerHTML = `
+   <div class='popUpHeader'>
+   <h1>${dynmicTitle}</h1>
+   <i class='bx bx-x'></i>
+   </div>
+   <ul>
+   <li>HTML</li>
+   <li>Bootstrap</li>
+   <li>Ruby on Rails</li>
+   </ul>
+${dynamicImage}
+   <p>             Lorem ipsum dolor sit amet
+    consectetur adipisicing elit.
```

Comment:
- [ ] You have done great so far :+1: :+1: But there are still some improvements you need to make to the UI.
- - As the previous reviewer pointed out, the pop-up for the desktop screens doesn't match the Figma design well. Looking at the Figma, you can see that the image covers about 60% of the pop-up while the text takes about 40%. This seems to be the reverse in your project. I'll suggest you update this. :smile: :wave: 
- - SCREENSHOT FIGMA:
![Screenshot from 2023-06-29 16-29-27](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/107632780/35e35c56-beb2-4d48-8c3e-a4d84550efc6)

- - YOUR DESIGN:
![Selection_072](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/107632780/7157c44d-a909-4e87-bc63-685f9e26fe3d)